server/src/main/java/com/cloud/uuididentity/UUIDManagerImpl.java

```diff
@@ -119,7 +119,9 @@ public boolean IsUuidFormat(String uuid) {
         }
         Identity identity = (Identity) this._entityMgr.findById(entityType, customId);
```

Comment:
@DaanHoogland when `entityType` is null, I think, it fails here (with NPE below stack), can you please check that.

Reply:
Yhe NPE would be thrown in the call to `findById()` @sureshanaparti @rhtyd 
```
    @Override
    public <T, K extends Serializable> T findById(Class<T> entityType, K id) {
        GenericDao<? extends T, K> dao = (GenericDao<? extends T, K>)GenericDaoBase.getDao(entityType);
        return dao.findById(id);
    }
```
I'm fine with applying the extra check in this case but I think it'll be unreachable code.

Reply:
@sureshanaparti , as shown above, this NPE is going to happen out of scope of this change anyway, in these kinds of bits of code. What do you suggest to do?

Reply:
@DaanHoogland my suggestion is if entityType is null, better throw exception as below

```
InvalidParameterValueException("Unknown entity type")
```

and  when identity is null (here entityType is not null), better throw exception as below

```
throw new InvalidParameterValueException(String.format("Unable to find UUID for id [%s] of type [%s]",
                                                                    customId, entityType.getSimpleName()));
```

Reply:
sounds good @sureshanaparti